docs/user-guide/build-environment.md

```diff
@@ -11,4 +11,10 @@
 * `ARGOCD_APP_SOURCE_REPO_URL` the repo's URL
 * `ARGOCD_APP_SOURCE_TARGET_REVISION` - the target revision from the spec, e.g. `master`.
 * `KUBE_VERSION` - the version of kubernetes
-* `KUBE_API_VERSIONS` = the version of kubernetes API
\ No newline at end of file
+* `KUBE_API_VERSIONS` = the version of kubernetes API
+
+> v2.3.0
```

Comment:
According to changes in `repository.go` kustomize is going to receive the same environment variables as other tools. I think this is correct. Can you please update the document accordingly?